### DISCUSSION TITLE
How to use libsecret when installed with vcpkg

### DISCUSSION THREAD

**Discussion by illera88** [microsoft/vcpkg#27579] [Q&A] [answered]
When you install a package you normally get the commands need to be used in your CMakelist.txt to find the installed package but there is nothing when libsecret is installed. Example:
```
sqlite3 provides CMake targets:

    # this is heuristically generated, and may not be correct
    find_package(unofficial-sqlite3 CONFIG REQUIRED)
    target_link_libraries(main PRIVATE unofficial::sqlite3::sqlite3)

```

How should I look for `libsecret` in CMake if installed using vcpkg?

**Comment by JackBoosY** [comment, marked as answer]
It seems that the libsecret upstream doesn't provide the method so you can only use `find_path` and `find_library` to use this port manually.

**Comment by Osyotr** [comment]
You can use pkgconfig:
```
find_package(PkgConfig REQUIRED)
pkg_check_modules(libsecret-1 REQUIRED IMPORTED_TARGET libsecret)
target_link_libraries(my_target PRIVATE PkgConfig::libsecret)
```
